### ISSUE TITLE
✅ 🔗 🚀 Feature Request: Icon list: Icon default alignment (Duplicate of #16056) - [ED-9265]

### ISSUE THREAD

**Issue by ra-ther** [elementor/elementor#18420]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Since few versions, the component Icon list align the item icon to the center by default. I presume it was a visual criteria, but following the UX reading order conventions, the correct aligment to an icon (or another symbol) should be at the top. 

Rather, I don't consider it a bug but a bad default behaviour. It should be on top by defaul but it should be customizable, because sometimes we use the component with visual purposes. 

### Describe the solution you'd like

It could be resolved with a new setting in the design panel called icon alignment, setting for instance a new CSS class like `.icon-center` or `.icon-bottom` to set `align-items: center` or `align-items: start`. 

Since I noticed the change, I coded a CSS snippet solution quickly that I place in every new elementor website I made. I hope it can help:

```
/*Elementor List icon fix*/

.elementor-widget .elementor-icon-list-item, 
.elementor-widget .elementor-icon-list-item a {
    align-items: start;
}

span.elementor-icon-list-icon {
    margin-top: 5px;
}
```

### Describe alternatives you've considered

As I said, I presume the icon centering criteria was based on visual purposes. I have think about why we do that and what other alternatives can we found in the component library. The most similar component to this use case is Icon Box / left align icon, with richer settings as title plus description or icon style. But when you have some items to place with the Icon box component it can be heavy to maintain because is not reliable at all.

Maybe a new component between icon list and list icon with left align components should solve this. This new one could have some features from Icon box like optional icon customizations in the design panel and keep it as simple as an icon list in content panel.

Or maybe the Icon list component is correct and we need a new one designed for just reading and listing (not visual) purposes.

**Comment by perks98** [comment]
I think this has been reported before, but yes 100% agree. It use to align at the top but after a few major changes they changed it to be central with no options to change. I think this has been a problem for a least a year.

**Comment by PixelDesigns** [comment]
This is a very old (1-2 year ago) UX problem and very annoying. Two orientation options (top, center) are required.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.12**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
